vortexasdk/endpoints/eia_forecasts_result.py

```diff
@@ -0,0 +1,41 @@
+from typing import List
+
+import pandas as pd
+
+from vortexasdk.api import EIAForecast
+from vortexasdk.api.search_result import Result
+from vortexasdk.logger import get_logger
+from vortexasdk.result_conversions import create_dataframe, create_list
+
+logger = get_logger(__name__)
+
+
+class EIAForecastResult(Result):
+    """Container class that holds the result obtained from calling the `EIAForecasts` endpoint."""
+
+    def to_list(self) -> List[EIAForecast]:
+        """Represent EIAForecast data as a list."""
+        # noinspection PyTypeChecker
+        return create_list(super().to_list(), EIAForecast)
+
+    def to_df(self, columns=None) -> pd.DataFrame:
+        """
+        Represent EIA forecasts as a `pd.DataFrame`.
+
+        # Arguments
+            columns: The EIA forecasts features we want in the dataframe. Enter `columns='all'` to include all features.
```

Comment:
```suggestion
            columns: The EIA forecasts columns we want in the dataframe. Enter `columns='all'` to include all columns.
```

Reply:
In the world of datascience, the term features normally refers to the input features of a model. In this sense of the word, we're better off using 'column' rather than features, as these aren't input features.